torch/_inductor/codecache.py

```diff
@@ -1118,7 +1118,7 @@ class CompiledFxGraph:
     mutated_inputs: Set[str]
     mutated_input_idxs: Set[int]
     constants: Dict[str, torch.Tensor]
-    torchbind_constants: Dict[str, torch._C.ScriptObject]
+    torchbind_constants: Dict[str, Union[FakeScriptObject, torch._C.ScriptObject]]
```

Comment:
is CompiledFXGraph the key to the graph cache or the value?

Reply:
I think the value?